kafkacat.c

```diff
@@ -311,32 +311,28 @@ static void producer_run (FILE *fp, char **paths, int pathcnt) {
 
         } else {
                 /* Read messages from input, delimited by conf.delim */
+                void *state = getdelims_init(conf.delim);
+                int sep_len = strlen(conf.key_delim);
                 while (conf.run &&
-                       (len = getdelim(&sbuf, &size, conf.delim, fp)) != -1) {
```

Comment:
`getdelim` is now dead code

Reply:
it's a glibc library function, but yes, the win32 version could be removed.